.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
I think we could exclude all hidden folders, not just this one but thinking about it again, I don't think it's a big deal.

Reply:
ad5179285